test/functional/test-document-submit.js

```diff
@@ -97,19 +97,20 @@ describe('test-document-submit onDocumentFormSubmit_', () => {
     evt.target.setAttribute('method', 'post');
     expect(() => onDocumentFormSubmit_(evt))
         .to.throw(/Only XHR based \(via action-xhr attribute\) submissions/);
-    expect(preventDefaultSpy.callCount).to.equal(1);
+    expect(preventDefaultSpy).to.have.been.called;
```

Comment:
preventDefault is now being called multiple times which doesn't really matter all I care about that it was called.